### PR TITLE
Add highlight CSS (closes #9)

### LINKED ISSUE #9: enable syntax highlighting in fenced snippets
It seems that rouge ships automatically with Jekyll but for whatever reason the site does not currently render any syntax highlighting. I assume this is to do with the current css and/or jekyll plugins setup.

For example, the snippet

```cpp
#include <iostream>

int main()
{
    std::cout << "Hello, world!" << std::endl;
    return 0;
}
```

does not have any syntax highlighting on the site either when fenced with ``` or with the liquid {% highlight [lang] %} <snippet> {% endhighlight %} tags.

### PR DESCRIPTION
**Pull request by mniip**
This is a copy of the `native` theme from pygments

### CODE REVIEW

File: _includes/extra_styles/highlight.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
+.highlight .hll { background-color: #404040; }
+.highlight { border-radius: 4px; background: #202020; color: #d0d0d0; }
+.highlight :is(.c, .ch, .cm, .cpf, .c1) { color: #ababab; font-style: italic; }
+.highlight .err { color: #a61717; background-color: #e3d2d2; }
+.highlight :is(.esc, .g, .l, .n, .o, .x, .p, .ld, .ni, .nl, .nx, .py, .pm) { color: #d0d0d0; }
+.highlight :is(.k, .kc, .kd, .kn, .kr, .kt, .nt, .ow) { color: #6ebf26; font-weight: bold; }
+.highlight .cp { color: #cd2828; font-weight: bold; }
```
Comment (rattus-rattus-rattus) [inline _includes/extra_styles/highlight.css:7]: i think #cd2828 is a little too dark to parse easily on the dark background; perhaps #e74242?

**Review by rattus-rattus-rattus — COMMENTED** [review]
not too sure about the red color but otherwise lgtm

File: _includes/extra_styles/highlight.css (hunk 1 above)
Comment (mniip) [reply _includes/extra_styles/highlight.css:7]: It's only used for "preprocessor directives" and it's bold, I think it's fine:
image

Comment (rattus-rattus-rattus) [reply _includes/extra_styles/highlight.css:7]: you're right. For some reason it was rendering quite differently on my browser earlier (but now appears as above?). review marked approved.

**Review by rattus-rattus-rattus — APPROVED** [review]

**Pull request merged**